include/openssl/ec.h

```diff
@@ -269,6 +269,13 @@ OPENSSL_EXPORT int EC_POINT_point2cbb(CBB *out, const EC_GROUP *group,
                                       point_conversion_form_t form,
                                       BN_CTX *ctx);
 
+// EC_POINT_point2bn converts an |EC_POINT| to |BIGNUM.| Returns the pointer to the
+// BIGNUM supplied or NULL on error.
```

Comment:
Does the user need to allocate ret/ctx?

Reply:
I updated this description to clarify the use of ret/ctx in the latest commit.